src/mixins/people-roles-mixin.ts

```diff
@@ -317,10 +318,12 @@ export default class PeopleRolesMixin extends Vue {
   protected hasValidRoles (): boolean {
     return (
       this.validNumCompletingParty &&
-      this.validMinimumIncorporators &&
-      this.validMinimumDirectors &&
+      this.validNumIncorporators &&
+      this.validNumDirectors &&
       this.validDirectorCountry &&
       this.validDirectorProvince &&
+      this.validApplicantPerson &&
+      this.validApplicantOrg &&
```

Comment:
New checks for applicants. These local getters (computeds) return True if the rule isn't defined, so this works fine for non-restoration filings.